common/config/azure-pipelines/integration-pr-validation.yaml

```diff
@@ -10,267 +10,268 @@ pr:
   drafts: false
   branches:
     include:
-    - master
-    - release/*
+      - master
+      - release/*
 
 variables:
-- group: iModel.js non-secret config variables
-- group: iModel.js Integration Test Users
+  - group: iModel.js non-secret config variables
+  - group: iModel.js Integration Test Users
 
 jobs:
-- job: Build
-  condition: succeeded()
-  strategy:
-    matrix:
-      "Node 10.17.0":
-        Node_Version: 10.17.0
-      "Node 10.x":
-        Node_Version: 10.x
-      "Node 12.x":
-        Node_Version: 12.x
+  - job: Build
+    condition: succeeded()
+    strategy:
+      matrix:
+        "Node 10.17.0":
+          Node_Version: 10.17.0
+        "Node 10.x":
+          Node_Version: 10.x
+        "Node 12.x":
+          Node_Version: 12.x
 
-  pool:
-    vmImage: ubuntu-latest
-  steps:
-  - checkout: self
-    clean: all
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+      - checkout: self
+        clean: all
 
-  - template: templates/setup-integration-users.yaml
+      - template: templates/setup-integration-users.yaml
 
-  - task: NodeTool@0
-    displayName: 'Use Node $(Node_Version)'
-    inputs:
-      versionSpec: '$(Node_Version)'
-      checkLatest: true
+      - task: NodeTool@0
+        displayName: "Use Node $(Node_Version)"
```

Comment:
Concerned about all of these incidental changes?

Reply:
They should be fine, sorry about that... Switched machines and it has auto-formatting turned on for the Azure pipelines Extension in VSCode.